### ISSUE TITLE
Josefin Sans 

### ISSUE THREAD

**Issue by davelab6** [google/fonts#1217]
> seems to contain a bug with tops of letters coming out container.
>
> -https://twitter.com/realredranger/status/910117019269427200

**Comment by thlinard** [comment]
Duplicate of #1045?

**Comment by m4rc1e** [comment]
@thlinard @davelab6 I'll look into it this week. 

During the redesign, we had to enable UseTypoMetrics in order to avoid clipping. I'm guessing all these parts have some involvement in this issue. I will investigate it further this week.

**Comment by laerm0** [comment]
Looks like WinAscent and WinDescent need to be tweaked. Shall I go ahead and fix source or is this a hotfix?

**Comment by laerm0** [comment]
Well, the Typo values need recalculation as well...

Are we trying to keep some kind of metric compatibility with the previous version of Josefin Sans?

**Comment by laerm0** [comment]
So I recalculated the metrics and played around with their balance for a while and got something that I think looks pretty good. @thlinard, @m4rc1e: opinions? This is the regular.

![screen region 2017-09-26 at 14 47 59](https://user-images.githubusercontent.com/10780161/30878098-cc9a4594-a2c9-11e7-9aa4-254fefc7c58a.png)

**Comment by laerm0** [comment]
Hinting is meh, sadly. :(

**Comment by laerm0** [comment]
Windows looks better.

![screen region 2017-09-26 at 14 50 20](https://user-images.githubusercontent.com/10780161/30878172-0d9685f8-a2ca-11e7-84e0-84170835ede8.png)

**Comment by thlinard** [comment]
Yes, hinting is meh, at the very least! One of the many problems remaining to be fixed in this typeface, probably.
But if you succeed to fix the clipping problem, it's always better than nothing. However, the leading seems too loose in your last example.

**Comment by laerm0** [comment]
Leading is a little loose, but that's linegap set to zero and the metrics are calculated so as not to clip and synced across the family. Unless we cheat a little, I'm afraid that's the most rational leading. The x-height is exactly half of the caps so I think it's okay if it's a little generous.

As to users now possibly getting some text reflow, that's a different story...

**Comment by davelab6** [comment]
> Looks like WinAscent and WinDescent need to be tweaked. Shall I go ahead and fix source or is this a hotfix?

@laerm0 confirmed offline that this family was updated by Thomas Jockin, so the .glyphs source file was ready to rebuild with a fix on this, and he's done it. 

Currently this is blocked by @m4rc1e shipping his differ'nator; when that is done, the next action is  then to add a before/after comparison here, so that he or I can make a decision on if this fix is good to publish.

**Comment by dolbex** [comment]
Did this ever get solved?

Below are some screenshots with line-height at 1 and no vertical (top/bottom) padding.

![Screen Shot 2020-03-26 at 3 47 52 PM](https://user-images.githubusercontent.com/954095/77690263-65ed5500-6f79-11ea-9626-9da826f82c87.png)
![Screen Shot 2020-03-26 at 3 46 45 PM](https://user-images.githubusercontent.com/954095/77690265-6685eb80-6f79-11ea-9809-17db1aa37171.png)
![Screen Shot 2020-03-26 at 3 41 14 PM](https://user-images.githubusercontent.com/954095/77690266-6685eb80-6f79-11ea-98d3-967d976ce814.png)
